tests/tests.rs

```diff
@@ -21,6 +22,10 @@ fn run_cbindgen(
     cpp_compat: bool,
     style: Option<Style>,
 ) {
+
```

Comment:
rustfmt wants to drop this newline: https://travis-ci.org/github/eqrion/cbindgen/builds/670071458